GetIntoTeachingApi/Adapters/OrganizationServiceAdapter.cs

```diff
@@ -23,7 +23,7 @@ public string CheckStatus()
         {
             try
             {
-                _client.GetMyUserId();
+                _client.GetEntityDisplayName("contact");

```

Comment:
An obvious question :-) but does the CRM not have a healthcheck endpoint which is lightweight and just a basic ping ?

Reply:
If it does its sadly not available through the SDK we're using 😞 I checked their WebAPI docs and couldn't find anything there either so I don't think one exists/is publicly visible